test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatCollection.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [CollectionDefinition("NuGet XPlat Test Collection")]
```

Comment:
Following the guidelines by xunit - https://xunit.github.io/docs/shared-context.html